llvm/utils/lit/tests/unique-output-file.py

```diff
@@ -0,0 +1,28 @@
+# Check that lit will not overwrite existing result files when given
```

Comment:
Something I've been encouraging people to do with tests I've reviewed is use `##` (or equivalent) for "true" comments, to help distinguish them from `RUN` and `CHECK` style lines. I feel it makes the test file easier to read.

Reply:
Done.